trouble-shooting.md

```diff
@@ -7,6 +7,25 @@ Known bugs and issues:
  * (Android) Tapping an alert in the notification centre will sometimes not result in `onNotification` being called [issue 281](https://github.com/zo0r/react-native-push-notification/issues/281)
  * (Android) Not all local notification features are supported yet (PRs welcome)
  * (iOS) The OS can penalise your app for not calling the completion handler and will stop (or delay) sending notifications to your app. This will be supported from RN-0.38 [PR 227](https://github.com/zo0r/react-native-push-notification/pull/277)
+ * (Android and iOS) If local notifications only works with remote debugger enabled it could be due to the date that you are adding to the notification, [explanation](https://stackoverflow.com/a/41881765/8519917).
```

Comment:
Is it possible to precise that this issue occures when using string format ? it was not clear as first read 😉

Reply:
Changed!